[2025-04-23T02:26:13.364+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: mbta_ingestion_dag.load_to_snowflake scheduled__2025-04-23T02:20:00+00:00 [queued]>
[2025-04-23T02:26:13.369+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: mbta_ingestion_dag.load_to_snowflake scheduled__2025-04-23T02:20:00+00:00 [queued]>
[2025-04-23T02:26:13.369+0000] {taskinstance.py:2170} INFO - Starting attempt 1 of 2
[2025-04-23T02:26:13.376+0000] {taskinstance.py:2191} INFO - Executing <Task(PythonOperator): load_to_snowflake> on 2025-04-23 02:20:00+00:00
[2025-04-23T02:26:13.380+0000] {standard_task_runner.py:60} INFO - Started process 204 to run task
[2025-04-23T02:26:13.383+0000] {standard_task_runner.py:87} INFO - Running: ['***', 'tasks', 'run', 'mbta_ingestion_dag', 'load_to_snowflake', 'scheduled__2025-04-23T02:20:00+00:00', '--job-id', '356', '--raw', '--subdir', 'DAGS_FOLDER/mbta_ingestion_dag.py', '--cfg-path', '/tmp/tmpmem37qxa']
[2025-04-23T02:26:13.385+0000] {standard_task_runner.py:88} INFO - Job 356: Subtask load_to_snowflake
[2025-04-23T02:26:13.428+0000] {task_command.py:423} INFO - Running <TaskInstance: mbta_ingestion_dag.load_to_snowflake scheduled__2025-04-23T02:20:00+00:00 [running]> on host 34e11901efc2
[2025-04-23T02:26:13.484+0000] {taskinstance.py:2480} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='mbta_ingestion_dag' AIRFLOW_CTX_TASK_ID='load_to_snowflake' AIRFLOW_CTX_EXECUTION_DATE='2025-04-23T02:20:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2025-04-23T02:20:00+00:00'
[2025-04-23T02:26:13.485+0000] {mbta_ingestion_dag.py:35} INFO - ðŸš€ Launching Kafka consumer to process and load data into Snowflake...
[2025-04-23T02:26:13.486+0000] {connection.py:370} INFO - Snowflake Connector for Python Version: 3.6.0, Python Version: 3.9.18, Platform: Linux-6.10.14-linuxkit-aarch64-with-glibc2.36
[2025-04-23T02:26:13.486+0000] {connection.py:1171} INFO - This connection is in OCSP Fail Open Mode. TLS Certificates would be checked for validity and revocation status. Any other Certificate Revocation related exceptions or OCSP Responder failures would be disregarded in favor of connectivity.
[2025-04-23T02:26:14.000+0000] {mbta_kafka_consumer.py:45} INFO - âœ… Connected to Snowflake successfully.
[2025-04-23T02:26:14.000+0000] {mbta_kafka_consumer.py:220} INFO - ðŸš€ Starting MBTA Kafka Consumer...
[2025-04-23T02:26:14.002+0000] {conn.py:395} INFO - <BrokerConnection client_id=kafka-python-2.1.5, node_id=bootstrap-0 host=kafka:9092 <connecting> [IPv4 ('172.18.0.4', 9092)]>: connecting to kafka:9092 [('172.18.0.4', 9092) IPv4]
[2025-04-23T02:26:14.020+0000] {conn.py:615} INFO - <BrokerConnection client_id=kafka-python-2.1.5, node_id=bootstrap-0 host=kafka:9092 <checking_api_versions_recv> [IPv4 ('172.18.0.4', 9092)]>: Broker version identified as 2.6
[2025-04-23T02:26:14.020+0000] {conn.py:456} INFO - <BrokerConnection client_id=kafka-python-2.1.5, node_id=bootstrap-0 host=kafka:9092 <connected> [IPv4 ('172.18.0.4', 9092)]>: Connection complete.
[2025-04-23T02:26:14.021+0000] {consumer.py:119} WARNING - group_id is None: disabling auto-commit.
[2025-04-23T02:26:14.022+0000] {subscription_state.py:182} INFO - Updating subscribed topics to: ('mbta_alerts', 'mbta_routes', 'mbta_stops', 'mbta_vehicles', 'mbta_predictions')
[2025-04-23T02:26:14.037+0000] {subscription_state.py:254} INFO - Updated partition assignment: [('mbta_alerts', 0), ('mbta_predictions', 0), ('mbta_routes', 0), ('mbta_vehicles', 0), ('mbta_stops', 0)]
[2025-04-23T02:26:14.039+0000] {conn.py:395} INFO - <BrokerConnection client_id=kafka-python-2.1.5, node_id=1 host=kafka:9092 <connecting> [IPv4 ('172.18.0.4', 9092)]>: connecting to kafka:9092 [('172.18.0.4', 9092) IPv4]
[2025-04-23T02:26:14.039+0000] {conn.py:456} INFO - <BrokerConnection client_id=kafka-python-2.1.5, node_id=1 host=kafka:9092 <connected> [IPv4 ('172.18.0.4', 9092)]>: Connection complete.
[2025-04-23T02:26:14.040+0000] {conn.py:936} INFO - <BrokerConnection client_id=kafka-python-2.1.5, node_id=bootstrap-0 host=kafka:9092 <connected> [IPv4 ('172.18.0.4', 9092)]>: Closing connection. 
[2025-04-23T02:26:19.028+0000] {conn.py:936} INFO - <BrokerConnection client_id=kafka-python-2.1.5, node_id=1 host=kafka:9092 <connected> [IPv4 ('172.18.0.4', 9092)]>: Closing connection. 
[2025-04-23T02:26:19.038+0000] {fetcher.py:735} INFO - Fetch to node 1 failed: Cancelled: <BrokerConnection client_id=kafka-python-2.1.5, node_id=1 host=kafka:9092 <connected> [IPv4 ('172.18.0.4', 9092)]>
[2025-04-23T02:26:19.038+0000] {connection.py:718} INFO - closed
[2025-04-23T02:26:19.155+0000] {connection.py:724} INFO - No async queries seem to be running, deleting session
[2025-04-23T02:26:19.278+0000] {mbta_kafka_consumer.py:241} INFO - âœ… Snowflake Connection Closed.
[2025-04-23T02:26:19.279+0000] {python.py:201} INFO - Done. Returned value was: None
[2025-04-23T02:26:19.297+0000] {taskinstance.py:1138} INFO - Marking task as SUCCESS. dag_id=mbta_ingestion_dag, task_id=load_to_snowflake, execution_date=20250423T022000, start_date=20250423T022613, end_date=20250423T022619
[2025-04-23T02:26:19.342+0000] {local_task_job_runner.py:234} INFO - Task exited with return code 0
[2025-04-23T02:26:19.357+0000] {taskinstance.py:3280} INFO - 0 downstream tasks scheduled from follow-on schedule check
